draft-ietf-taps-impl.md

```diff
@@ -776,9 +776,18 @@ Connection Properties. A Connection can also generate error events in the form o
 
 The set of Connection Properties that are supported for setting and getting on a Connection are described in {{I-D.ietf-taps-interface}}. For
 any properties that are generic, and thus could apply to all protocols being used by a Connection, the Transport Services implementation should store the properties
-in storage common to all protocols, and notify all protocol instances in the Protocol Stack whenever the properties have been modified by the application. {{!RFC8303}} and {{!RFC8304}} offer guidance on how to do this for TCP, MPTCP, SCTP, UDP and UDP-Lite; see {{specific-protocol-considerations}} for a description of a back-tracking method to find the relevant protocol primitives using these documents.
+in storage common to all protocols, and notify the Protocol Stack as a whole whenever the properties have been modified by the application. {{!RFC8303}} and {{!RFC8304}} offer guidance on how to do this for TCP, MPTCP, SCTP, UDP and UDP-Lite; see {{specific-protocol-considerations}} for a description of a back-tracking method to find the relevant protocol primitives using these documents.
 For Protocol-specific Properties, such as the User Timeout that applies to TCP, the Transport Services implementation only needs to update the relevant protocol instance.
 
+Some Connection Properties might apply to multiple protocols within a Protocol Stack. Depending on the specific property,
+it might be appropriate to apply the property across these multiple protocols simultaneously, or else only apply to one.
```

Comment:
"these multiple protocols" sounds like all of the protocols in the protocol stack to me, but it could be multiple, but still a subset of the protocols in the stack so perhaps:

... apply the property across multiple protocols simultaneously, or else only apply it to one protocol.

Reply:
```suggestion
it might be appropriate to apply the property across multiple protocols simultaneously, or else only apply it to one protocol.
```